config/cache.php

```diff
@@ -44,7 +44,7 @@
 
         'file' => [
             'driver' => 'file',
-            'path' => storage_path('framework/cache/data'),
```

Comment:
What is the `/data` path removed? This might require removing the old cache if the storage path doesn't change?

Reply:
Mistake! Thanks for catching that.